lib/hued/light.rb

```diff
@@ -0,0 +1,138 @@
+
+module Hued
+
+  class Color
+    attr_reader :config
+    attr_reader :brightness, :hue, :saturation, :x, :y
```

Comment:
switch this (and related attrs) to be accessors, and then add a `save` method that will push the updated openstruct to the hub